### PR TITLE
[API] add api tag to applying promotions with expiration date

### PR DESCRIPTION
**Pull request by arti0090**
| Q               | A
| --------------- | -----
| Branch?         | master
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | no
| License         | MIT

### CODE REVIEW

File: features/promotion/receiving_discount/applying_promotion_with_expiration_date.feature [hunk 1]
```diff
@@ -10,28 +10,28 @@ Feature: Applying promotion with an expiration date
...
-    @ui
+    @ui @api
     Scenario: Receiving a discount from a promotion which has already started
         Given this promotion has started yesterday
         When I add product "PHP T-Shirt" to the cart
         Then my cart total should be "$90.00"
         And my discount should be "-$10.00"
 
-    @ui
+    @ui @apig
```
Comment (oallain) [inline features/promotion/receiving_discount/applying_promotion_with_expiration_date.feature:34]: Hello,
What is `@apig` ?

```suggestion
    @ui @api
```

Comment (arti0090) [reply features/promotion/receiving_discount/applying_promotion_with_expiration_date.feature]: 🤭  @oallain its when i try to write git but im still in phpstorm window 😄

Comment (oallain) [reply features/promotion/receiving_discount/applying_promotion_with_expiration_date.feature]: :joy: it happens.

**Review by oallain — APPROVED** [review]

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Thanks, @arti0090! :1st_place_medal:

**Pull request merged**